db/sg_replicate_cfg.go

```diff
@@ -972,6 +987,9 @@ func (m *sgReplicateManager) AddReplication(replication *ReplicationCfg) error {
 		if exists {
 			return false, base.ErrAlreadyExists
 		}
+		if replication.ClusterUUID == "" {
```

Comment:
Is it possible to add a test to verify correct behaviour on CAS retry on concurrent update of the cluster (i.e. we don't end up with two replications with different cluster UUIDs)

Reply:
As discussed, this isn't a blocker to merge this, as replications existing with different cluster UUIDs doesn't break the underlying functionality.